### ISSUE TITLE
Risk Register creation

### ISSUE THREAD

**Issue by NickCorcoran** [BCDevOps/developer-experience#4762]
**Describe the issue**
Keeping track of STRA recommendations and general security risks in a centralized place is valuable for visibility and updates.
OCIO ISB has a sharepoint site where they have their own Risk Register as a SharePoint list.  Eventually this may morph into an enterprise risk management system tied to the Digital STRA system in Service Now.  Setting up a similarly configured list would put us in best position for integration in future.

**Additional context**
Add any other context, attachments or screenshots

**How does this benefit the users of our platform?**

**Definition of done**
- [x] Discussion w/ OCIO ISB
- [x] Obtain list settings for Risk Register
- [ ] Create list in sharepoint site
- [ ] Import items from Excel Spreadsheet
- [ ] Enter recommendations from STRA SoARs that are not in current list

*Future task to review line items and provide updated status

**Comment by NickCorcoran** [comment]
Have asked OCIO ISB if they can export the list definition to import (so we don't have to recreate manually)

**Comment by AvneetKahlon** [comment]
waiting on ServiceNow(digital STRA)access

**Comment by NickCorcoran** [comment]
Will move to closed and create a new ticket for entry of missing SoAR recommendations.  Great work on getting the core list setup!
